Orange/widgets/data/owrandomize.py

```diff
@@ -87,6 +90,8 @@ def _set_scope_label(self):
     def set_data(self, data):
         self.data = data
         self.unconditional_apply()
+        self.info.set_input_summary(str(len(data))) if data else \
```

Comment:
That is actually an assignment expression, but result is not assigned to anything. You can e.g. create a new variable and use it to set the summary.
`text = str(len(data)) if self.data else self.info.NoInput`